lib/csdl2markdown.js

```diff
@@ -335,14 +336,54 @@ module.exports.csdl2markdown = function (filename, csdl) {
                 + '|' + descriptionInTable(p));
         });
 
+        return lines;
+    }
+
+    
+     /**
+     * Table of applicable terms of structured type
+     * @param {object} type Structured type
+     * @return {Array} Array of strings containing Markdown lines
+     */
+    function applicableTermsTable(type) {
+        const lines = [];
+
+        const tLines = applicableTermLines(type);
+        if (tLines.length > 0) {
+            lines.push('');
+            lines.push('Notable terms');
+            lines.push('');
+            lines.push('Term|Type');
```

Comment:
```suggestion
            lines.push('Term|Link to Definition');
```

Or we could make the `@Core.Description` etc. the link and not repeat the substring. Then maybe use a simple bullet list instead of a table with only one column.

Reply:
Updated to a markdown format that looks like:
```
- [Description](Org.OData.Core.V1.md#Description)
- [LongDescription](Org.OData.Core.V1.md#LongDescription)
```